src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -261,7 +279,21 @@ def post(self, request):
                     self.open_resolve_dialog(data, group, integration)
                     defer_attachment_update = True
         except client.ApiError as e:
-            return self.api_error(e, action_type, logging_data)
+
+            unlinking_url = build_unlinking_url(
+                integration.id, group.organization.id, user_id, channel_id, response_url
+            )
+
+            if e.status_code == 403:
+                text = UNLINK_IDENTITY_MESSAGE.format(
```

Comment:
@NisanthanNanthakumar Nit: I think the call to `build_unlinking_url` should be inside this if condition since we don't use it anywhere else.